src/metaschema/oscal_metadata_metaschema.xml

```diff
@@ -40,22 +71,255 @@
+                        </description>
+                    </role>
+                </builtins>
+                -->
+                <remarks>
+                    <!-- TODO: roles only appear in metadata. Referenced in the document and linked instances. -->
+                    <!-- TODO: discuss built-in roles. -->
+                    <p>Permissible values to be determined closer to the application (e.g. by a receiving authority).</p>
+                    <p>OSCAL has defined a set of standardized roles for consistent use in OSCAL documents. This allows tools consuming OSCAL content to infer specific semantics when these roles are used. These roles are documented in the specific contexts of their use (e.g., responsible-party, responsible-role). When using such a role, it is necessary to define these roles in this list, which will then allow such a role to be referenced.</p>
+                </remarks>
+            </define-assembly>
+            <define-assembly name="location" max-occurs="unbounded">
+                <formal-name>Location</formal-name>
+                <description>A physical point of presence, which may be associated with people, organizations, or other concepts within the current or linked OSCAL document.</description>
                 <group-as name="locations" in-json="ARRAY"/>
-            </assembly>
-            <assembly ref="party" max-occurs="unbounded">
+                <define-flag name="uuid" as-type="uuid" required="yes">
+                    <formal-name>Location Universally Unique Identifier</formal-name>
+                    <description>A <a href="/concepts/identifier-use/#machine-oriented">machine-oriented</a>, <a href="/concepts/identifier-use/#globally-unique">globally unique</a> identifier with <a href="/concepts/identifier-use/#cross-instance">cross-instance</a> scope that can be used to reference this defined location elsewhere in <a href="/concepts/identifier-use/#scope">this or other OSCAL instances</a>. The locally defined <em>UUID</em> of the <code>location</code> can be used to reference the data item locally or globally (e.g., from an importing OSCAL instance). This UUID should be assigned <a href="/concepts/identifier-use/#consistency">per-subject</a>, which means it should be consistently used to identify the same subject across revisions of the document.</description>
+                </define-flag>
+                <model>
+                    <define-field name="title" as-type="markup-line">
+                        <formal-name>Location Title</formal-name>
+                        <description>A name given to the location, which may be used by a tool for display and navigation.</description>
+                    </define-field>
+                    <assembly ref="address">
+                        <remarks>
+                            <p>The physical address of the location.</p>
+                        </remarks>
+                    </assembly>
```

Comment:
This use to be `min-occurs="1"`, but we wanted to relax this to allow for non-physical locations. The remarks should indicate this.

```suggestion
                    <assembly ref="address">
                        <remarks>
                            <p>The physical address of the location, which will provided for physical locations. Virtual locations can omit this data item.</p>
                        </remarks>
                    </assembly>
```